src/test/Withdrawals.t.sol

```diff
@@ -17,12 +18,85 @@ contract WithdrawalTests is DelegationTests {
         uint96 nonce;
     }
 
+    address public registryCoordinator = address(uint160(uint256(keccak256("registryCoordinator"))));
+    ServiceManagerMock public serviceManager;
+    StakeRegistryHarness public stakeRegistry;
+    StakeRegistryHarness public stakeRegistryImplementation;
```

Comment:
Flagging the use of this as a potential merge conflict with the branch that removes all middleware contracts -- see https://github.com/Layr-Labs/eigenlayer-contracts/pull/260 or more specifically https://github.com/Layr-Labs/eigenlayer-contracts/blob/209e6c85d760b7913bfa286e59ab0189714219f9/src/test/EigenLayerTestHelper.t.sol#L7-L9
@stevennevins tagging you here for visibility
My thought is that this is probably OK to merge rn, we'll just have to iron out the issue afterwards.